src/pages/HomePage.js

```diff
@@ -0,0 +1,41 @@
+import styled from 'styled-components/macro'
+import Alert from '../components/Alert/Alert'
+import FilterForm from '../components/FilterForm/FilterForm'
+import Recipe from '../components/Recipe/Recipe'
+import SearchBar from '../components/SearchBar/SearchBar'
+
+export default function HomePage({
+  onRecipeSearch,
+  text,
+  dietLabels,
+  allergiesLabels,
+  cuisineTypes,
+  onFindClicked,
+  recipes,
+}) {
+  return (
+    <>
```

Comment:
You could delete this fragment. Your `PageLayout` is the one parent element necessary.